.github/workflows/docker.yml

```diff
@@ -7,6 +7,14 @@ on:
   pull_request:
     paths:
```

Comment:
Maybe for the less familiar with Actions add a comment at the top like:

"Composable and reusable GitHub action workflows" or something similar + the link to the docs section

Reply:
Sure, added this info both for reuseable workflows and for composite actions.